app/serializers/dependents_serializer.rb

```diff
@@ -60,7 +60,7 @@ def current_and_pending_decisions(diaries)
   end
 
   def dependency_decisions(diaries)
-    decisions = diaries && diaries[:dependency_decs]
+    decisions = diaries.is_a?(Hash) && diaries[:dependency_decs]
```

Comment:
What is diaries is nil?
```suggestion
    decisions = diaries&.is_a?(Hash) && diaries[:dependency_decs]
```